test/helpers/defaults.go

```diff
@@ -33,6 +36,26 @@ func GetDefaultDataPlaneImage() string {
 	return _defaultDataPlaneImage
 }
 
+// GetDefaultDataPlaneImagePrevious returns the default data plane image with the previous version.
+func GetDefaultDataPlaneImagePrevious() string {
```

Comment:
This will tell more about what the previous means. 
```suggestion
func GetDefaultDataPlaneImagePreviousMinor() string {
```